guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -53,12 +69,12 @@ To fork the devdocs repository on Github, do the following:
 
 1.  Clone the repository into your development environment and then start writing and committing your changes. Optionally, create a branch first if you plan to work on multiple changes.
 
-1.  Build the devdocs site locally using Jekyll by following the instructions in our [README](https://github.com/magento/devdocs/blob/develop/README.md){:target="\_blank"}.
+1.  Build the devdocs site locally using Jekyll by following the instructions in our [README](https://github.com/magento/devdocs/blob/develop/README.md){:target="_blank"}.
```

Comment:
Follow the instructions in.... to build the DevDocs site locally using Jekyll."

Reply:
"Create or log into..." not "in to"

Reply:
edited